source/includes/administration/activity_log.md

```diff
@@ -0,0 +1,197 @@
+      "eventCode": "users.created",
+      "userFirstname": "User",
+      "userOrganizationName": "System",
+      "category": "SYSTEM",
+      "updated": "2021-02-02T14:12:43.709Z",
+      "status": "SUCCESS",
+      "username": "username"
+    }
+  ]
+}
+```
+
+An array of activities with the following attributes are returned.
+
+Activity Attributes | &nbsp;
+------------------- | -----------
+`id`<br/>*UUID* | The id of the activity.
+`correlationId`<br/>*UUID* | The correlation id of the activity.
+`eventCode`<br/>*string* | The human readable code for the event.
+`category`<br/>*string* | The category the event belongs to.
+`status`<br/>*string* | The status of the event.
+`created`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) the activity was created.
+`updated`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) the activity was created.
+`userId`<br/>*UUID* | The id of the user who generated the activity.
+`username`<br/>*string* | The username of the user who generated the activity.
+`userFirstname`<br/>*string* | The first name of the user who generated the activity.
+`userLastname`<br/>*string* | The first name of the user who generated the activity.
+`userEmail`<br/>*string* | The email of the user who generated the activity.
+`userOrganizationId`<br/>*UUID* | The id of the organization of the user who generated the activity.
+`userOrganizationName`<br/>*string* | The name of the organization of the user who generated the activity.
+`organizationId`<br/>*UUID* | The id of the organization of activity belongs to.
+`organizationName`<br/>*string* | The name of the organization of activity belongs to.
+`requesterIp`<br/>*string* | The ip address of the requester.
+`eventContext`<br/>*string* | The escaped json event context.
+
+<!------------------- LIST ACTIVITY CODES -------------------->
+
+### List activity codes
+
+`GET /activity_log/codes`
+
+Retrieves a list of activity codes. Optional parameters `organizationId` and `include_subs` can be specified to narrow the list of activities being returned.
+
+Optional Query Parameters  | &nbsp;
+-------------------------- | -----------
+`organizationId`<br/>*UUID* | The organization id for which we want to get the activity codes from. When not specified, the activity codes for the organization the current user is associated to will be returned.
+`include_subs`<br/>*boolean* | If we want to include sub-organizations activity codes.
+
+```shell
+# Retrieve quotas list
+curl "https://cloudmc_endpoint/api/v1/activity_log/codes" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "SYSTEM": [
+      "res_commitments.update",
+      "users.autocreated",
+      "environment_members.purge",
+      "custom_field.deleted",
+      "organizations.provision",
+      "trials.converted_to_billable",
+      ...
```

Comment:
Here I added ellipses to indicate that there can be many and that is just a sample. Is it a good idea or should I just remove it? 🤔

Reply:
I dont mind it but we should make it a comment and change the language to `js` instead of `json`
```suggestion
      // ...
```